Library/Formula/tarantool.rb

```diff
@@ -19,22 +19,12 @@ def install
     args = std_cmake_args
     args << "-DCMAKE_INSTALL_MANDIR=#{doc}"
     args << "-DCMAKE_INSTALL_SYSCONFDIR=#{etc}"
+    args << "-DCMAKE_INSTALL_LOCALSTATEDIR='#{var}'"
```

Comment:
Why is `'#{var}'` being passed in `''` whereas the rest are passed directly, out of interest?